src/main/drivers/sdcard_spi.c

```diff
@@ -174,7 +225,7 @@ static uint8_t sdcard_waitForNonIdleByte(int maxDelay)
  */
 static uint8_t sdcard_sendCommand(uint8_t commandCode, uint32_t commandArgument)
 {
-    const uint8_t command[6] = {
+    uint8_t command[6] = {
```

Comment:
Why is this not const any more?

Reply:
Because it's use in the segment definition would discard the qualifier thus:

```
./src/main/drivers/sdcard_spi.c:242:14: warning: initialization discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
  242 |             {command, NULL, sizeof (command), false, NULL},
```